src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -106,12 +119,16 @@ public ValidationResult validate() {
 		ValidationResult validation =  new ValidationResult();
 		final Matcher matcher = pattern.matcher(webhookUrl());
 
+		if(backlogSize() < 0 || backlogSize() > MAX_BACKLOG_MESSAGES) {
```

Comment:
Why do we impose a maximum number of messages for the backlog size?

Reply:
just so we don't overload the slack client with messages, that it can't handle. we can tune the paramter, as we do more performance testing.